### PR TITLE
[processing/en] Add Language Processing

### PR DESCRIPTION
**Pull request by phonethantko**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: processing.html.markdown [hunk 1]
```diff
@@ -0,0 +1,423 @@
+---
+language: "Processing"
```
Comment (divayprakash) [inline processing.html.markdown]: Please correct this to -
`language: processing`

Comment (divayprakash) [inline processing.html.markdown]: Please correct this to `language: processing`

File: processing.html.markdown [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+Processing is easy to learn and is particularly useful to create multimedia contents (even in 3D) without
+having to type a lot of codes. It is so simple that you can read through the code and get a rough idea of
+the program flow.  
+However, that does not apply when you introduce external libraries, packages and even your own classes.
+(Trust me! Processing projects can get real humongous...)  
+
+## What's Next?
+
+Here, I have compiled some useful resources:
```
Comment (divayprakash) [inline processing.html.markdown]: Please change this to `Some useful resources:`

**Comment by divayprakash** [comment]
@phonethantko Thank you for the contribution and welcome to the community!

**Comment by divayprakash** [comment]
@vendethiel @adambard This looks good to me but I'd like your review before merging it in.

**Comment by divayprakash** [comment]
@phonethantko Please make the changes as I requested and I'll merge this in ASAP. Thank you for the contribution!

**Comment by phonethantko** [comment]
@divayprakash Updated 👍 Let me know if there's anything you want to change.

**Comment by phonethantko** [comment]
@divayprakash Updated 👍 Let me know if there's anything you want to change

**Comment by divayprakash** [comment]
@phonethantko Thank you for the contribution and welcome to the community!

**Pull request merged**